### PR TITLE
chore: revise issue template

### PR DESCRIPTION
**Pull request by seolaoh**
# Description

Added expected behavior and revised the issue template to use for general purpose.

### CODE REVIEW

**Review by 0xHansLee — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/issue_template.yaml [hunk 1]
```diff
@@ -20,36 +20,42 @@ body:
...
   - type: textarea
-    id: logs
+    id: additional-context
     attributes:
-      label: Relevant log output
-      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
-      render: shell
\ No newline at end of file
+      label: Additional context
+      description: Please copy and paste any relevant log output. Or add any other context or screenshots about the feature request.
```
Comment (chokobole) [inline .github/ISSUE_TEMPLATE/issue_template.yaml]: Please put blank here!

**Review by kangsorang — APPROVED** [review]
lgtm

**Review by Pangssu — APPROVED** [review]
LGTM

**Review by chokobole — APPROVED** [review]
LGTM

**Pull request merged**